full/src/main/java/apoc/date/DateExpiry.java

```diff
@@ -18,15 +18,15 @@
 @Extended
 public class DateExpiry {
 	@Procedure(mode = Mode.WRITE, deprecatedBy = "apoc.ttl.expireAtInstant")
-	@Description("CALL apoc.date.expire(node,time,'time-unit') - expire node at specified time by setting :TTL label and `ttl` property")
+	@Description("CALL apoc.ttl.expire(node,time,'time-unit') - expire node at specified time by setting :TTL label and `ttl` property")
```

Comment:
```suggestion
	@Description("CALL apoc.ttl. expireAtInstant(node,time,'time-unit') - expire node at specified time by setting :TTL label and `ttl` property")
```

Fai attenzione per favore, non avere fretta